examples/helpers/android/app/build.gradle.in

```diff
@@ -29,7 +29,11 @@ android {
                         "@EDGE_SDK_EXAMPLE_NETWORK_PROTOCOL_JAR@",
                         "@EDGE_SDK_EXAMPLE_LEVELDB_DIR@",
                         // required for finding installed packages in sysroot
-                        "-DCMAKE_FIND_ROOT_PATH_MODE_PACKAGE=BOTH"
+                        "-DCMAKE_FIND_ROOT_PATH_MODE_PACKAGE=BOTH",
+                        "-DBOOST_ROOT=@BOOST_ROOT@",
+                        "-DBoost_INCLUDE_DIR=@BOOST_ROOT@",
+                        "-DBoost_LIBRARY_DIR=@BOOST_ROOT@",
+                        "-DANDROID_PLATFORM=android-28"
```

Comment:
```suggestion
                        "-DANDROID_PLATFORM=@ANDROID_PLATFORM@"
```
We shouldn't hardcode value here to 28 platform, cause user can build for other versions